library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -242,6 +243,25 @@ object ConstructorIo {
         hiddenFacets?.forEach { hiddenFacet ->
             encodedParams.add(Constants.QueryConstants.FMT_OPTIONS.format(Constants.QueryConstants.HIDDEN_FACET).urlEncode() to hiddenFacet.urlEncode())
         }
+        fmtOptions?.forEach { (key, value) ->
+            when (value) {
+                is List<*> -> {
+                    value.forEach{ fmtOptionsValue ->
+                        if (fmtOptionsValue is String) {
+                            encodedParams.add(Constants.QueryConstants.FMT_OPTIONS.format(key).urlEncode() to fmtOptionsValue.urlEncode())
+                        } else if (fmtOptionsValue is Number) {
+                            encodedParams.add(Constants.QueryConstants.FMT_OPTIONS.format(key).urlEncode() to fmtOptionsValue.toString())
+                        }
+                    }
+                }
+                is String -> {
```

Comment:
Do we need to handle the case where it's a boolean or would we just pass a string 'true' or 'false' here?